src/Symfony/Component/Form/Button.php

```diff
@@ -122,9 +122,8 @@ public function getParent()
      *
      * This method should not be invoked.
      *
-     * @param int|string|FormInterface $child
-     * @param null                     $type
-     * @param array                    $options
+     * @param int|string|\FormInterface $child
```

Comment:
wrong `\`, I stop here :) the fixer seems to consider that classes are all coming from the global namespace

Reply:
@TomasVotruba Can you have a look at this issue?

Reply:
Fixed allong with `&$value` references.

I discovered one more error to resolve